syntaxes/weidu-tp2.tmLanguage.yml

```diff
@@ -4786,7 +4793,7 @@ repository:
     patterns:
       - include: '#weidu-vars'
       - include: '#ielib-constants'
-      - match: (%)((\w*)(%[#!\-\w]+%)(\w*))(%)
+      - match: (%)((\w*)(%[#!<>=\-\w]+%)(\w*))(%)
```

Comment:
Someone's using <, >, = in variable names? Do you have links?

Reply:
> Someone's using <, >, = in variable names? Do you have links?

Probably only me, will remove if it is a problem...

Reply:
So do you have links?

Reply:
> So do you have links?

No, I don't. Removed.